### PR TITLE
Contribution guide.

### PR DESCRIPTION
**Pull request by michalchudziak**
I've prepared some basic contribution guide - there are some places which need a word from @toddw or @cybrass. It's just a proposal so feel free to change anything :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+* **Most important: Look around.** Match the style you see used in the rest of the project. This includes formatting, naming things in code, naming things in documentation.
+* Add trailing commas,
+* 2 spaces for indentation (no tabs)
+
+#### JavaScript
+
+* We are using extended `eslint-config-airbnb` - make sure your code doesn't result in any lint errors.
+* We are also using [FlowType](https://flowtype.org/) for static type checking - make sure your code doesn't produce any flow errors.
+* No `Object` or `Function` types are allowed (except for edge cases) - make sure all types have detailed description.
```
Comment (ferrannp) [inline CONTRIBUTING.md]: I'd add `any` type here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+* When writing CLI we use `CommonJS` to handle import/export - please do not mix them with ES6 imports (only exception is importing types).
```
Comment (ferrannp) [inline CONTRIBUTING.md]: I propose:

> When writing CLI we use `CommonJS` to handle import/export - please do not mix them with ES6 imports. The only exception are Flow types where we use ES6 imports.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# Contributing to GlueStick
+
+GlueStick is one of TrueCar's open source projects that is build to speed up the development process internally in [truecar.com](https://truecar.com) and also help a larger audience reduce the time needed to build apps. It allows the developer to create and deploy enterprise-grade React and Redux application right from your command line.
+
+### Pull Requests
+
+TrueCar team will be monitoring for pull requests.
```
Comment (grabbou) [inline CONTRIBUTING.md]: Not sure I get that sentence?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+# Contributing to GlueStick
+
+GlueStick is one of TrueCar's open source projects that is build to speed up the development process internally in [truecar.com](https://truecar.com) and also help a larger audience reduce the time needed to build apps. It allows the developer to create and deploy enterprise-grade React and Redux application right from your command line.
```
Comment (grabbou) [inline CONTRIBUTING.md]: Try `Gluestick is an open source project sponsored by TrueCar (link) that uses it to speed up the...

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+2. **Describe your test plan in your commit.** If you've added code that should be tested, add tests!
+3. If you've changed APIs, update the documentation.
+4. Ensure tests, lints and flow-check pass on *Circle CI*.
+5. Ensure that *Node Security* check passed.
+6. Squash your commits (`git rebase -i`).
+   one intent alongs with one commit makes it clearer for people to review and easier to understand your intention.
+
+### Contributor License Agreement (CLA)
+
+TODO: @toddw
```
Comment (toddw) [inline CONTRIBUTING.md:21]: I'm working on this now

Comment (toddw) [reply CONTRIBUTING.md:21]: Talking to people about setting up a CLA bot like https://github.com/clabot/clabot. Anyone have other suggestions or experience with cla bot?

Comment (michalchudziak) [reply CONTRIBUTING.md:21]: Unfortunately not.

**Review by ferrannp — APPROVED** [review]

**Pull request merged**